src/systems/filecoin_vm/actor/actor.id

```diff
@@ -116,3 +117,5 @@ type ActorSubstateCID ipld.CID
 //         NextID      ActorID
 //     }
 // }
+
+type BalanceTableHAMT {addr.Address: TokenAmount}
```

Comment:
Please separate this (the implementation) out into a new package outside the core VM package, perhaps in the markets alongside the actors that will use it.

Reply:
I don't have a strong preference for where it should go, but will note that it's needed by both the StoragePowerActor and the StorageMarketActor (at least; and possibly other modules in the future).

Reply:
agree, may make sense (organizationally) to store alongside the rewards actor or burnt funds actor